kinappserver/views_public.py

```diff
@@ -1435,3 +1435,61 @@ def get_discovery_apps():
     os_type = get_user_os_type(user_id)
 
     return jsonify(get_discovery_apps(os_type))
+
+
+
+@app.route('/contact-us', methods=['POST'])
+def contact_support_endpoint():
+    """create a new ticket in zendesk"""
+    import requests, json
+    payload = request.get_json()
+    try:
+        category = payload.get('category', None).replace(" ", "_")
+        sub_category = payload.get('sub_category', None).replace(" ", "_")
+        name = payload.get('name', None)
+        email = payload.get('email', None)
+        description = payload.get('description', None)
+        user_id = payload.get('user_id', None)
+        platform = payload.get('platform', None)
+        version = payload.get('version', None)
+        debug = payload.get('debug', None)
+        if None in (category, sub_category, name, email, description, user_id, platform, version):
+            raise InvalidUsage('bad-request')
+
+        log.debug("user_id: %s - submitted a ticket" % user_id)
+    except Exception as e:
+        print(e)
+        raise InvalidUsage('bad-request')
+    else:
+        from .models.user import create_ticket
+        if create_ticket(name,email,category,sub_category,description,user_id,platform,version,debug):
+            return jsonify(status='ok')
+    
+    raise InvalidUsage('bad-request')
```

Comment:
I think you meant to `return jsonify(status='failed')` here

Reply:
changed it, thought about it and it's easier to handle "passed" or "failed", for the client it doesn't matter the reason.

Reply:
doesn't look changed... 
Note that on the feedback_endpoint you do `return jsonify(status='failed')`